docs/pipelines.md

```diff
@@ -397,37 +398,57 @@ spec:
       Timeout: "0h1m30s"
 ```
 
-### Configuring execution results at the `Task` level
+## Using `Results`
 
-Tasks can emit [`Results`](tasks.md#storing-execution-results) while they execute. You can
-use these `Results` values as parameter values in subsequent `Tasks` within your `Pipeline`
-through [variable substitution](variables.md#variables-available-in-a-pipeline). Tekton infers the
-`Task` order so that the `Task` emitting the referenced `Results` values executes before the
-`Task` that consumes them. 
+Tasks can emit [`Results`](tasks.md#emitting-results) when they execute. A Pipeline can use these
+`Results` for two different purposes:
 
-In the example below, the result of the `previous-task-name` `Task` is declared as `bar-result`:
+1. A Pipeline can pass the `Result` of a `Task` in to the `Parameters` of another.
+2. A Pipeline can itself emit `Results` and include data from the `Results` of its Tasks.
+
+### Passing one Task's `Results` into the `Parameters` of another
+
+Sharing `Results` between `Tasks` in a `Pipeline` happens via
+[variable substitution](variables.md#variables-available-in-a-pipeline) - one `Task` emits
+a `Result` and another receives it as a `Parameter` with a variable such as
+`$(tasks.<task-name>.results.<result-name>)`.
+
+When one `Task` receives the `Results` of another, there is a dependency created between those
+two `Tasks`. In order for the receiving `Task` to get data from another `Task's` `Result`,
+the `Task` producing the `Result` must run first. Tekton enforces this `Task` ordering
+by ensuring that the `Task` emitting the `Result` executes before any `Task` that uses it. 
+
+In the snippet below, a param is provided its value from the `commit` `Result` emitted by the
+`checkout-source` `Task`. Tekton will make sure that the `checkout-source` `Task` runs
+before this one.
 
 ```yaml
 params:
   - name: foo
-    value: "$(tasks.previous-task-name.results.bar-result)"
+    value: "$(tasks.checkout-source.results.commit)"
 ```
 
 For an end-to-end example, see [`Task` `Results` in a `PipelineRun`](../examples/v1beta1/pipelineruns/task_results_example.yaml).
 
-## Configuring execution results at the `Pipeline` level
+### Emitting `Results` from a `Pipeline`
+
+A `Pipeline` can emit `Results` of its own for a variety of reasons - an external
+system may need to read them when the `Pipeline` is complete, they might summarise
```

Comment:
A sentence on how this system can read these results might help. Maybe:

"These results are made available in the `PipelineRun.Status.PipelineResults` field via the API".

Reply:
Added a note below,

> After a `Pipeline` has executed the `PipelineRun` will be populated with the `Results` emitted by the `Pipeline`. These will be written to the `PipelineRun's`
`status.pipelineResults` field.